app/src/store/actionCreators.js

```diff
@@ -0,0 +1,81 @@
+import { actionTypes } from './actionTypes';
+
+const globalUrl = 'http://localhost:4000';
+
+function getMovies(dispatch, getState){

```

Comment:
`const getMovies = () =>  (dispatch, getState) => {`

I believe and usage as dispatch(getMovies())

Reply:
Done